README.md

```diff
@@ -35,5 +35,24 @@ Besides the standard package, extra tooling (e.g., testing, formatting, linting,
 pip install -e ".[dev]"
 ```
 
+You can also use the included devcontainer, which preinstalls dependencies including:
+- GDAL (for `rasterio`) and 
+- `libgtk-3-dev` (for `cv2`)
+Use the "Reopen in container" command in VSCode or as a GitHub codespace.
+
 ## CLI
 Upon installation the `fsdproc` command will be available. View its help with `fsdproc --help`.
+
+## Cylc
+To run the `cylc` workflow with the test data, run:
+```bash
+cylc stop ebseg/*       # stops any currently running workflows
+cylc install . -n ebseg # installs the current version of the workflow
+cylc play ebseg         # runs the workflow
+cylc tui ebseg          # opens the text user interface
+```
+
+or on one line:
+```bash
+cylc stop ebseg/* ; cylc install . && cylc play ebseg && cylc tui ebseg
```

Comment:
```suggestion
cylc stop ebseg/*; cylc clean ebseg/*; cylc install . && cylc play ebseg && cylc tui ebseg
```

Reply:
Without the cylc clean, it should just start a new run of the `ebseg` flow. In practice, when a job hangs on Oscar, I had issues with being able to run a new run of a prior named workflow and had to add the cylc clean. Maybe it's not needed here, just something to keep in mind for Oscar testing.